README.md

```diff
@@ -181,12 +181,12 @@ If you have not done so, install Ruby. Here’s a helpful guide on [how to best
 
 Once you have installed Ruby, clone this repository to your machine. Then, navigate to it using Terminal or your preferred command line interface. Follow the steps below to run the site from your machine. **If you’re on Windows, don’t forget to run your CLI as an admin.**
 
-### First time install
+### First time installation
 
 1. Run `npm install`
 2. Run `bundle install`
 
-### Start-up
+### Launch application
```

Comment:
“Application” is pushing it a little… The command would start a local server that's running the website—“Launch server and website” may describe it but seems lengthy, too.

Reply:
Maybe we can stick with "Open" or "Launch" and the specifics will be underneath anyway?

Reply:
“Install” and “Launch”?

Reply:
I like "Install" and "Launch". I think keeping it vague here is okay and that there is an implication to installation that it is initial or the first time. @j9t Are we good to go with this?

Reply:
Sure, let’s do this! If you can update the PR, I’ll approve.